projects/storefrontlib/src/cms-components/myaccount/update-profile/components/update-profile-form.component.ts

```diff
@@ -1,7 +1,6 @@
 import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { FormBuilder, Validators } from '@angular/forms';
 import { Title, User } from '@spartacus/core';
-import { FormUtils } from '../../../../shared/utils/forms/form-utils';
 
 @Component({
```

Comment:
Breaking changes to document:
- changed output submited to submitted 
- removed isNotValid

Reply:
Done